.github/ISSUE_TEMPLATE/chore.yml

```diff
@@ -14,10 +14,20 @@ body:
     id: what
     attributes:
       label: What should be done?
-    validations:
-      required: true
+      description: Additional context you would like to provide.
   - type: textarea
     id: where
     attributes:
       label: Where?
       description: Include a permalink if possible.
+  - type: textarea
+    id: dod
+    attributes:
+      label: Definition of Done
+      description: What should be done to consider this issue done?
+      value: |
+        - [ ] compatibility tests added
+        - [ ] documentation updated
+        - [ ] anything else?
```

Comment:
```
Unless the issue explicitly states otherwise, the following things are always in the scope:

* Tests.
  See contributing documentation for general discussion about unit and integration tests.
* Small spot refactorings.
  Larger refactorings should be in a separate issue with minimal behavior changes.
* Minor updates to existing documentation.
  Completely new documentation should be in a separate issue.
```

I think the current text fails to include unit tests, non-compat integration tests (compat tests are not always possible or preferable for non-CRUD commands),  and refactoring.

I also think we could expand our PR template to include those things